sorc/build_ufs.sh

```diff
@@ -50,5 +44,5 @@ if [ -d build ]; then
   rm -R build
 fi
 mkdir -p build && cd build
-cmake -DAPP=${APP} -DCCPP_SUITES=${CCPP_SUITES} ..
+cmake -DAPP=${APP} -DCCPP_SUITES=${CCPP_SUITES} -DUFS_GOCART="ON" ..
```

Comment:
`-DUFS_GOCART` is not an exposed option of the UFS.
The correct way to enable aerosols is using the `-DAPP=ATMAERO`.